tiledb/bioimg/converters/base.py

```diff
@@ -183,12 +184,9 @@ def from_tiledb(
 
         with tiledb.scope_ctx(config):
             vfs_use = output_path.startswith("s3://")
-            if vfs_use:
-                vfs = tiledb.VFS()
-                destination_uri = vfs.open(output_path, "wb")
-            else:
-                destination_uri = output_path
-
+            destination_uri = (
```

Comment:
Let's make this a parameter that can be passed from the user. Call it `scratch_space` or similar to distinguish from where the final location it.